app/views/admins/users/index.html.slim

```diff
@@ -7,7 +7,9 @@ div class="container home"
           tr
             td = t '.table.email_col'
             td = t '.table.last_visit_col'
+            td = t '.table.view_info_col'
         - @users.each do |user|
           tr
             td = user.email
             td = user.last_sign_in_at
+            td = link_to (fa_icon "eye"), admins_user_path(user.id), id: "link-to"
```

Comment:
adding the same id within a loop will render invalid HTML. I can have only one unique id per page. Assign an id to a `tr` element, that includes `user.id`